apps/meteor/tests/end-to-end/api/livechat/14-units.ts

```diff
@@ -409,4 +413,428 @@ import { IS_EE } from '../../../e2e/config/constants';
+		return request
+			.get(api(`livechat/department/${departmentId}`))
+			.set(credentials)
+			.expect('Content-Type', 'application/json')
+			.expect(200)
+			.expect((res) => {
+				expect(res.body).to.have.property('success', true);
+				expect(res.body).to.have.property('department');
+				expect(res.body.department).to.have.property('_id', departmentId);
+				expect(res.body.department).to.have.property('name', departmentName);
+				if (ancestor) {
+					expect(res.body.department).to.have.property('parentId', ancestor);
+					expect(res.body.department).to.have.property('ancestors').that.is.an('array').with.lengthOf(1);
+					expect(res.body.department.ancestors[0]).to.equal(ancestor);
+				}
+
+				if (ancestor === null) {
+					expect(res.body.department).to.have.property('parentId').that.is.null;
+					expect(res.body.department).to.have.property('ancestors').that.is.null;
+				}
+
+				if (ancestor === undefined) {
+					expect(res.body.department).to.not.have.property('parentId');
+					expect(res.body.department).to.not.have.property('ancestors');
+				}
+			});
+	};
+
+	describe('[POST] livechat/department', () => {
+		let departmentId: string;
+		let monitor1: Awaited<ReturnType<typeof createUser>>;
+		let monitor1Credentials: Awaited<ReturnType<typeof login>>;
+		let monitor2: Awaited<ReturnType<typeof createUser>>;
+		let monitor2Credentials: Awaited<ReturnType<typeof login>>;
+		let unit: IOmnichannelBusinessUnit;
+
+		const createDepartmentInUnit = (userCredentials: Credentials, departmentName: string, unitId: string) => {
+			return request
+				.post(api('livechat/department'))
+				.set(userCredentials)
+				.send({
+					department: { name: departmentName, enabled: true, showOnOfflineForm: true, showOnRegistration: true, email: 'bla@bla' },
+					departmentUnit: { _id: unitId },
+				})
+				.expect('Content-Type', 'application/json')
+				.expect(200)
+				.expect((res) => {
+					expect(res.body).to.have.property('success', true);
+					expect(res.body).to.have.property('department').that.is.an('object');
+					expect(res.body.department).to.have.property('name', departmentName);
+					expect(res.body.department).to.have.property('type', 'd');
+					expect(res.body.department).to.have.property('_id');
+					departmentId = res.body.department._id;
+				});
+		};
+
+		before(async () => {
+			monitor1 = await createUser();
+			monitor2 = await createUser();
+			await createMonitor(monitor1.username);
+			monitor1Credentials = await login(monitor1.username, password);
+			await createMonitor(monitor2.username);
+			monitor2Credentials = await login(monitor2.username, password);
+			unit = await createUnit(monitor1._id, monitor1.username, []);
+		});
+
+		after(async () => Promise.all([deleteUser(monitor1), deleteUser(monitor2), deleteUnit(unit)]));
+
+		afterEach(() => {
+			if (departmentId) {
+				return deleteDepartment(departmentId);
+			}
+		});
```

Comment:
I suggest keeping the deletion inside of each `it` creating it, otherwise, since it's not applied for all `it`s it could cause confusion on when it should be removed.